client/asset/dcr/dcr.go

```diff
@@ -979,7 +1015,7 @@ func (dcr *ExchangeWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin
 	if err != nil {
 		return nil, nil, 0, err
 	}
-	pkScripts := make([][]byte, 0, len(swaps.Contracts))
+	contractScripts := make([][]byte, 0, len(swaps.Contracts))
```

Comment:
Up on line 1000 (`dcr.log.Debugf("locking coins %v", coins)`) the message is different from btc's, which is "locking funding coins that were unlocked".  Would prefer these messages match.